api/webservice/Portal/BaseModule/Record.php

```diff
@@ -107,19 +108,25 @@ public function get()
 			foreach ($rawInventory as $row) {
 				$inventoryRow = [];
 				foreach ($inventoryFields as $name => $field) {
-					$inventoryRow[$name] = $field->getDisplayValue($row[$name]);
+					$inventoryRow[$name] = $field->getDisplayValue($row[$name], $row, true);
 				}
 				$inventory[] = $inventoryRow;
 			}
+			foreach ($inventoryFields as $name => $field) {
+				if ($field->isSummary()) {
+					$summaryInventory[$name] = \CurrencyField::convertToUserFormat($field->getSummaryValuesFromData($rawInventory), null, true);
```

Comment:
\CurrencyField::convertToUserFormat is deprecated
App\Fields\Currency::formatToDisplay